test/acceptance/test/add_tests.go

```diff
@@ -161,7 +171,7 @@ var _ = Describe("Weave GitOps Add App Tests", func() {
 		})
 	})
 
-	It("Test1 - Verify that gitops can deploy an app after it is setup with an empty repo initially", func() {
+	FIt("Test1 - Verify that gitops can deploy an app after it is setup with an empty repo initially", func() {
```

Comment:
Is it intentional that this test is focused on this test case?

Reply:
good catch